src/authorization/WebAclReader.ts

```diff
@@ -75,8 +102,7 @@ export class WebAclReader extends PermissionReader {
    * @param acl - Store containing all relevant authorization triples.
    * @param isAcl - If the target resource is an acl document.
    */
-  private async createPermissions(credentials: CredentialSet, acl: Store, isAcl: boolean):
-  Promise<PermissionSet> {
+  private async createPermissions(credentials: CredentialSet, acl: Store, isAcl: boolean): Promise<PermissionSet> {
```

Comment:
```suggestion
  private async extractPermissions(acl: Store, credentials: CredentialSet, isAclResource: boolean): Promise<PermissionSet> {
```
(plus comment on line 100)